gatsby-config.js

```diff
@@ -53,12 +59,13 @@ const getContentfulOptions = () => {
 // a server side variable into a runtime one. This variable is later used to determine which
 // branch of a split testing experiment we are on.
 process.env.GATSBY_GIT_BRANCH_NAME = process.env.BRANCH;
+process.env.GATSBY_NETLIFY_SITE_NAME = process.env.SITE_NAME;
 
 module.exports = {
   siteMetadata: {
     title: SITE_TITLE,
     description: 'Hosted, managed, enterprise Backstage',
-    siteUrl: 'https://roadie.io',
+    siteUrl: getSiteUrl(),
```

Comment:
This is not directly related to this fix but seems sensible anyway. On the preview site, the site URl is not roadie.io.